### PR TITLE
Feature/352 enable chosing between ball and stick and only stick

### PR DESCRIPTION
**Pull request by Michel-Heinz**
The pull request for the structure customizer is here. You can open the menu under view->Open Structure Customizer
You can save, delete, and load presets. The Default preset is loaded upon starting and can be overwritten.

## Summary by CodeRabbit

- **New Features**
	- Introduced a dialog in the GUI for customizing molecular structure appearance, enabling users to adjust ball and stick sizes, toggle between modes, control bond visibility, and save/load settings.

- **Enhancements**
	- Enhanced bond management in molecular structures, automatically adjusting based on structure presence.
	- Improved handling of cylinder dimensions and properties in the `drawer.py` file to enhance customization of molecular structures.

### CODE REVIEW

**Review by ab5424 — APPROVED** [review]

File: Settings/Structure/Default.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"stick_mode": false, "bonds": true, "ball_size": 0.9999999999999992, "stick_size": 1.0}
```
Comment (ab5424) [inline Settings/Structure/Default.json:1]: @Michel-Heinz That's an interesting value, but I guess it just has to be smaller than the stick size.

Comment (Michel-Heinz) [reply Settings/Structure/Default.json:1]: Ah nice, I did not see that. I think this is just double precision magic

**Comment by GereonFeldmann** [comment]
Nice feature, looks really good so far. I have four minor issues to address. 

1. The names of the settings are not case sensitive. Is this intended? 
2. You cannot change the settings without a molecule imported. Again, is this intended?
3. I think it would be good if we add a query, whether a setting should be deleted. This is not big of an issue atm. However, if we increases the variety of settings, this will be crucial.
4. The json for the settings is currently in a structures folder, I guess you intend to use several json for differing settings.
 I think we should actually put it in one. Imo, as this is in a dictionary style, it is still clearly arranged and you will not get lost   in several setting files.

**Comment by Michel-Heinz** [comment]
> Nice feature, looks really good so far. I have four minor issues to address.
> 
> 1. The names of the settings are not case sensitive. Is this intended?
> 2. You cannot change the settings without a molecule imported. Again, is this intended?
> 3. I think it would be good if we add a query, whether a setting should be deleted. This is not big of an issue atm. However, if we increases the variety of settings, this will be crucial.
> 4. The json for the settings is currently in a structures folder, I guess you intend to use several json for differing settings.
>    I think we should actually put it in one. Imo, as this is in a dictionary style, it is still clearly arranged and you will not get lost   in several setting files.

1 Interesting, i thought the would cas sensitive, I ll have a look
2 Never thoguht about that, but makes sense, how do you want to change something if you don t see what you are doing?
3 Already implemented, but not pushed
4 Thought about that decided not to do that, in order for user to exchange single setting files without much work :)

**Comment by Michel-Heinz** [comment]
@GereonFeldmann

**Pull request merged**